ElrondNetwork/elrond-go
# [DO NOT MERGE]Broken development

- added faulty return

How to test:
1. start a testnet with this branch (nu upgade, all in)
2. at epoch 4 metachain will stop producing blocks
3. restart the metachain nodes. The following error message should appear:
```
doBlockJob.createBlock                   error = missing header : GetMarshalizedHeaderFromStorage hash = 65706f63685374617274426c6f636b5fxxxxxxxx
```
